trio/_highlevel_socket.py

```diff
@@ -130,7 +130,7 @@ async def send_eof(self) -> None:
             with _translate_socket_errors_to_stream_errors():
                 self.socket.shutdown(tsocket.SHUT_WR)
 
-    async def receive_some(self, max_bytes: int | None = None):
+    async def receive_some(self, max_bytes: int | None = None) -> Any:
```

Comment:
This should just be `-> bytes` I think, since that's what `recv()` ultimately [should return](https://docs.python.org/3/library/socket.html#socket.socket.recv)?

Reply:
ah yes, thanks!